sdk/formrecognizer/Azure.AI.FormRecognizer/samples/Sample_AnalyzePrebuiltRead.md

```diff
@@ -0,0 +1,140 @@
+# Analyze with the prebuilt read model
+
+This sample demonstrates how to analyze textual elements, such as page words and lines, styles, and text language information from documents using the prebuilt read model.
+
+To get started you'll need a Cognitive Services resource or a Form Recognizer resource.  See [README][README] for prerequisites and instructions.
```

Comment:
nit: I think we should say just an FR resource since it's not under CS anymore. Not sure when the multi-key access will stop working (if it hasn't already)